### DISCUSSION TITLE
Scikits solver error with version 23.3

### DISCUSSION THREAD

**Discussion by ganmad** [pybamm-team/PyBaMM#2891] [Q&A] [answered]
Hi,

I tried using Scikits solver (both Dae and Ode) with latest version (23.3) and it throws an error that "ImportError: scikits.odes is not installed" while it has been installed. 

Here is the confirmation that it has been installed: 
---
pybamm_install_odes
/mnt/c/Users/ganesh/Documents/pybamm2/pybamm/pybamm/scikits_odes_setup.log
/home/ganmad/.local/lib was found in LD_LIBRARY_PATH.
--> Not updating venv activate or .bashrc scripts
Requirement already satisfied: scikits.odes in ./.tox/dev/lib/python3.8/site-packages (2.7.0)
Requirement already satisfied: scipy in ./.tox/dev/lib/python3.8/site-packages (from scikits.odes) (1.10.1)
Requirement already satisfied: numpy<1.27.0,>=1.19.5 in ./.tox/dev/lib/python3.8/site-packages (from scipy->scikits.odes) (1.24.2)

Here is the minimum example I tired:
---
import pybamm
model = pybamm.lithium_ion.DFN()

sksolver = pybamm.ScikitsDaeSolver()
sksolver2 = pybamm.ScikitsOdeSolver()
sim = pybamm.Simulation(model,solver=sksolver)
sim.solve([0, 3600])
sim.plot()

---
ImportError: scikits.odes is not installed

Am I missing anything? 

Regards,
Ganesh

**Comment by Saransh-cpp** [comment, marked as answer]
Hi, are you running your script within the `tox` environment?

**Reply by ganmad** [reply]
Hi, Yes. I installed pybamm with tox and activated the environment. I am using pybamm on WSL with  integrated VS code.

**Reply by Saransh-cpp** [reply]
That sounds weird. Could you see if the library is actually installed -
```
pip show scikits.odes
```

**Reply by ganmad** [reply]
Hi Sarnash, Thanks for your time. I realised that correct python kernel wasn't active on VS code, and now it is working fine after activating the environment. All looks good now.  Thanks.
